apps/user/components/tips/view.coffee

```diff
@@ -0,0 +1,48 @@
+Backbone = require 'backbone'
+Cookies = require 'cookies-js'
+
+analytics = require '../../../../lib/analytics.coffee'
+mediator = require '../../../../lib/mediator.coffee'
+
+module.exports = class TipView extends Backbone.View
+
+  events: 
+    'click': 'handleClick'
+    'click .js-close' : 'removeTip'
+
+  removeTip: (e) ->
+    e.preventDefault()
+    e.stopImmediatePropagation()
+    @setClosed()
+    @remove()
+
+  setClosed: ->
+    analytics.track.click "Block tip closed", id: @model.id
+    Cookies.set @model.id, true
+
+  handleClick: (e) ->
+    e.preventDefault()
+    e.stopPropagation()
+
+    @setClosed()
+
+    if trigger = @$el.data('trigger')
+      switch trigger
+        when "new:channel"
+          # jQuery to open the new channel dialog, and close when body is clicked
+          $dropdown = $('.js-channel-create .js-dropdown')
```

Comment:
Add a new `js-` selector to handle this. These on their own refer to very different actions.